### DISCUSSION TITLE
Breakout room 6 for Series 03

### DISCUSSION THREAD

**Discussion by mikatakahashi** [IBAHCM/RPiR#216] [Archive]
Mika Takahashi - mikatakahashi
Hettie Pike - HettiePike
Yanis Chrysostomakis - YanisChrys
Loyce Majige - LoyceMajige

**Comment by HettiePike** [comment]
My repo link - https://github.com/IBAHCM/HettiePikeSeries03.git

**Comment by YanisChrys** [comment]
my repo - https://github.com/IBAHCM/YanisChrysSeries03

**Comment by mikatakahashi** [comment]
my repo link - https://github.com/IBAHCM/MikaTakahashiSeries03

**Comment by LoyceMajige** [comment]
my repo link - https://github.com/IBAHCM/LoyceMajigeSeries03

**Comment by LoyceMajige** [comment]
Hey @YanisChrys , @mikatakahashi , @HettiePike 

I have push all my series03 practical to GitHub, if you can please check to see if there are issues and provide me feedback when you get some time. 

Magic thanks,
Loyce
